tools/tune/eq/mls_freq_resp.m

```diff
@@ -273,7 +273,7 @@ function remote_capture(fn, cfg, t)
 	fprintf('Channels         : %d\n', rec.nch);
 	fprintf('Calibration Data : %s\n', rec.cal);
 
-	if length(rec.cal) > 0
+	if ~isempty(rec.cal) > 0
```

Comment:
Should this be without last "> 0" ?

Reply:
This is check for not empty . i.e. length higher than 0. So replace length with function call not empty

Reply:
Isn't just `if ~isempty(rec.cal)` enough?